views/kanban-view/src/components/Entry/Entry.tsx

```
@@ -74,13 +61,13 @@ export default function Entry({
       );
     });
 
-    const titleName = entry.hasOwnProperty("name")
+    const titleName = entries[0].hasOwnProperty("name")
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)